config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
I'm not too set on the time format, but as @jmartin-sul and I discussed a bit previously, the goal was to present the elapsed time in a more user-friendly format than just hh:mm:ss. I assume that the person using this page, even in downstream applications, is most likely to be a librarian or other non-technical curator, and thus it would be nice to present the elapsed time in a friendly, easily-scannable format (which doesn't have to be exact).